LongWarOfTheChosen/Src/LW_LaserPack_Integrated/Classes/X2Item_LaserWeapons.uc

```diff
@@ -6,8 +6,8 @@
 //---------------------------------------------------------------------------------------
 class X2Item_LaserWeapons extends X2Item config(GameData_WeaponData);
 
-//�Variables�from�config�-�GameData_WeaponData.ini
-//�*****�Damage�arrays�for�attack�actions��*****
+//�Variables�from�config�-�GameData_WeaponData.ini
+//�*****�Damage�arrays�for�attack�actions��*****
```

Comment:
What's this?

Reply:
I do not know how this happened. Somehow my editor replaced the spaces with unprintable characters.

I'll get this fixed.

Reply:
This is fixed now. I'll make sure to keep an eye for this again.